### PR TITLE
Update managing-office-365-endpoints.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the way allow customers to block consumer services usinghttps://docs.microsoft.com/en-us/azure/active-directory/manage-apps/tenant-restrictions#blocking-consumer-applications

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6233

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/managing-office-365-endpoints.md [hunk 1]
```diff
@@ -202,9 +202,7 @@ If you're trying to use Office 365 and are finding third-party services aren't a
...
-  
-Keep in mind that blocking access to the Microsoft consumer services alone won't prevent the ability for someone on your network to exfiltrate information using an Office 365 tenant or other service.
+The  tenant restrictions feature now supports blocking the use of all Microsoft consumer applications (MSA apps) such as OneDrive, Hotmail, and Xbox.com. This uses a separate header to the login.live.com endpoint. For more details, [Use tenant restrictions to manage access to SaaS cloud applications](/azure/active-directory/manage-apps/tenant-restrictions#blocking-consumer-applications)
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/managing-office-365-endpoints.md]:
```suggestion
The tenant restrictions feature now supports blocking the use of all Microsoft consumer applications (MSA apps) such as OneDrive, Hotmail, and Xbox.com. This uses a separate header to the login.live.com endpoint. For more details, see [Use tenant restrictions to manage access to SaaS cloud applications](/azure/active-directory/manage-apps/tenant-restrictions#blocking-consumer-applications),
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/enterprise/managing-office-365-endpoints.md [hunk 2]
```diff
@@ -202,9 +202,7 @@ If you're trying to use Office 365 and are finding third-party services aren't a
...
-  
-Keep in mind that blocking access to the Microsoft consumer services alone won't prevent the ability for someone on your network to exfiltrate information using an Office 365 tenant or other service.
+The tenant restrictions feature now supports blocking the use of all Microsoft consumer applications (MSA apps) such as OneDrive, Hotmail, and Xbox.com. This uses a separate header to the login.live.com endpoint. For more details, see [Use tenant restrictions to manage access to SaaS cloud applications](/azure/active-directory/manage-apps/tenant-restrictions#blocking-consumer-applications),
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/managing-office-365-endpoints.md]:
```suggestion
The tenant restrictions feature now supports blocking the use of all Microsoft consumer applications (MSA apps) such as OneDrive, Hotmail, and Xbox.com. This uses a separate header to the login.live.com endpoint. For more details, see [Use tenant restrictions to manage access to SaaS cloud applications](/azure/active-directory/manage-apps/tenant-restrictions#blocking-consumer-applications).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more minor edit please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pandrew1 Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@pandrew1 Gentle reminder

**Comment by yogkumgit** [comment]
@tiburd Please help us and follow with the stakeholder who can merge this PR. Thanks

**Pull request merged**